apps/etl-func/src/func.ts

```diff
@@ -34,6 +60,43 @@ const main = async (config: Config) => {
     methods: ["GET"],
     route: "health",
   });
+
+  const messageIngestion = async (documents: unknown[]) => {
+    const messagesToSend: MessageEvent[] = [];
```

Comment:
I think this array can be populated in a functional way, we should try to make all immutable!